README.md

```diff
@@ -52,7 +52,7 @@ If you are eager to get started, jump to the [Installation](#installation) and [
 
 ## Versions supported and requirements
 
-This extension has been tested with CKAN 2.8 and 2.9.
+This extension has been tested with CKAN 2.9.
```

Comment:
Can we put in a note on which version they should use if still on CKAN 2.8?

Reply:
Eh, I'd leave that to upstream. Maybe if and when we're merge compatible we can address that.